haiku-apps/bescreencapture/bescreencapture-2.1.recipe

```diff
@@ -0,0 +1,72 @@
+SUMMARY="A screen recorder utility"
+DESCRIPTION="BeScreenCapture is a screen recorder utility for Haiku.
+It allows you to record what happens on your screen, then save it \
+to any media format supported in Haiku.
+BeScreenCapture can record either the entire screen, or just a section you \
+select."
+
+SUMMARY_inputfilter="Shortcut handler for BeScreenCapture"
+DESCRIPTION_inputfilter="Input Server Addon for BeScreenCapture. Allows the \
+user to launch BeScreenCapture and start/stop recording using a keyboard \
+combination (CTRL-COMMAND-SHIFT + R)."
+
+HOMEPAGE="https://github.com/jackburton79/bescreencapture"
+COPYRIGHT="2014-2016 Stefano Ceccherini"
+LICENSE="BSD (3-clause)
+	MIT
```

Comment:
For LICENSE (and COPYRIGHT) it is recommended to put the closing quotes to the right of the last line, instead of putting it in a separate line.
You can have a look at [line 11 of bescreencapture-1.9.2.recipe](https://github.com/haikuports/haikuports/blob/56a7c925628a95e4fed51239c6f160ecad4b0248/haiku-apps/bescreencapture/bescreencapture-1.9.2.recipe#L11) to see the recommended layout for the LICENSE variable.